security/rbac/delta_configs/kafka-rest.properties.delta

```diff
@@ -1,3 +1,4 @@
+zookeeper.connect=localhost:2181
```

Comment:
looks like this was missing in 5.3

Reply:
@omkreddy IIUC, `zookeeper.connect` is not required because the properties file already has `bootstrap.servers=localhost:9092` (see below).  Plus the demo works without this line.  Can you please confirm, and if so, remove this line?

Reply:
@ybyzek kafka-rest still depends on ZK for licence validation. So we require `zookeeper.connect` config when used with confluent security extensions. kafka-rest server startup will fail without this config. 

> @omkreddy Thanks for the PR, LGTM. Since you are here, could you also change the listener name in `server.properties.delta` from RBAC to TOKEN please? We can do it in a separate PR if that is easier.

@rajinisivaram Thanks for the review. updated the PR.

Reply:
@omkreddy thanks for the explanation.  The default must already be `localhost:2181` because the demo works without setting that value.

Given that it's required but the default works, can you please add a comment above the line, something like: `# Configure connection to ZooKeeper, required for license validation of the Confluent security extensions`?